CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 *Please add new entries at the top.*
 
 1. Bumped deployment target to iOS 11, tvOS 11, watchOS 4, macOS 10.13, per Xcode 14 warnings
+1. Explicitly declare `APPLICATION_EXTENSION_API_ONLY`
```

Comment:
Cool, maybe mention that this is specific to the `CocoaPods` installation?

Reply:
Done!